sale_commission/models/res_partner.py

```diff
@@ -38,16 +40,18 @@ class ResPartner(models.Model):
     agent_type = fields.Selection(
         selection=[("agent", "External agent")], string="Type", required=True,
         default="agent")
-    commission = fields.Many2one(
+    commissions = fields.Many2many(
         string="Commission", comodel_name="sale.commission",
+        relation="agent_commission_rel",
+        column1="partner_id", column2="commission_id",
         help="This is the default commission used in the sales where this "
              "agent is assigned. It can be changed on each operation if "
              "needed.")
     settlement = fields.Selection(
-        selection=[("monthly", "Monthly"),
-                   ("quaterly", "Quarterly"),
-                   ("semi", "Semi-annual"),
-                   ("annual", "Annual")],
+        selection=[(sc.PERIOD_MONTH, "Monthly"),
```

Comment:
This is better to directly declare the full list with the tuples as a variable, like made in other places along Odoo.